SpaceCadetPinball/winmain.cpp

```diff
@@ -180,7 +180,7 @@ int winmain::WinMain(LPCSTR lpCmdLine)
 			if (!gfr_display)
 			{
 				auto plt = static_cast<ColorRgba*>(malloc(1024u));
-				auto pltPtr = &plt[10];
+				auto pltPtr = &plt[10]; // first 10 entries are system colors hardcoded in display_palette()
```

Comment:
As designed, also applies to palette from partman.

Reply:
Yes, this one was also just intended as a real comment to avoid somebody else having to look this up again.  :)